requirements-dev.txt

```diff
@@ -111,6 +111,10 @@ gitdb==4.0.9
     # via gitpython
 gitpython==3.1.27
     # via bandit
+greenlet==1.1.3
```

Comment:
The changes in `requirements.txt` and `requirements-dev.txt` should also be reverted.

Reply:
have reverted this too now :)